crates/bevy_ecs/src/system/query.rs

```diff
@@ -488,6 +488,33 @@ where
         }
     }
 
+    /// Gets the result of a single-result query.
+    ///
+    /// Assumes this query has only one result and panics if there are no or multiple results.
+    /// If you want to handle the error case yourself you can use the [`Self::try_single`] variant.
```

Comment:
```suggestion
    /// Use [`Self::try_single`] to handle the error cases explicitly
```
I don't think this suggestion is necessarily better, but I do think that this sentence could be improved

Reply:
I like it. Done.